copernicus-publications.csl

```diff
@@ -255,7 +255,14 @@
             <else-if type="article-journal" match="any">
               <group delimiter=", " prefix=" ">
                 <text variable="title"/>
-                <text variable="container-title-short"/>
+                <choose>
```

Comment:
You can just go with `<text variable="container-title" form="short"/>` which is equivalent to the entire conditional